seed_data/management/commands/seed_db.py

```diff
@@ -289,10 +294,12 @@ def handle(self, *args, **options):
             # recreate_index() is run afterwards to do this indexing in bulk.
             with mute_signals(post_save):
                 fake_programs = deserialize_program_data_list(program_data_list)
-                fake_course_runs = CourseRun.objects.filter(
-                    course__program__description__contains=FAKE_PROGRAM_DESC_PREFIX
-                ).all()
-                fake_user_count = deserialize_user_data_list(user_data_list, fake_course_runs)
+                fake_user_count = deserialize_user_data_list(user_data_list, fake_programs)
+
+            # Handle FA programs
+            fake_financial_aid_programs = filter(lambda program: program.financial_aid_availability, fake_programs)
+            create_tiers(fake_financial_aid_programs, int(options["tiers"]))
```

Comment:
If `seed_db` now runs the `create_tiers` command as well, could you update the `README` file to indicate that running `manage.py create_tiers` is no longer necessary?